test/array-buffer.js

```diff
@@ -100,17 +100,6 @@ test(
 	new Uint8Array(Buffer.from(`${fixtureString}${fixtureString[0]}`)).buffer,
 );
 
-test('handles streams larger than arrayBuffer max length', async t => {
-	t.timeout(BIG_TEST_DURATION);
-	const chunkCount = Math.floor(BufferConstants.MAX_LENGTH / CHUNK_SIZE * 2);
-	const chunk = Buffer.alloc(CHUNK_SIZE);
-	const maxBufferChunks = Array.from({length: chunkCount}, () => chunk);
-	const {bufferedData} = await t.throwsAsync(setupArrayBuffer(maxBufferChunks));
-	t.is(new Uint8Array(bufferedData)[0], 0);
-});
-
-const CHUNK_SIZE = 2 ** 16;
```

Comment:
Node 22 increased the maximum size of `Buffer` and `ArrayBuffer` to 75GB. This is too high to be able to run this test without either crashing the process or making the test run for too long.

Thankfully, we run a similar test but for the other methods `getStream()`, etc. so they should be covering that case.